src/generate/module/schema.json

```diff
@@ -64,6 +63,11 @@
       "description": "Flag to indicate if a dir is created.",
       "default": false
     },
+    "common": {
+      "type": "boolean",
+      "description": "Flag to control if a common file is created.",
```

Comment:
Consider extending the description to: 
"Flag to control if a common file is created. The common file exports components, providers and routes common for both Web and NativeScript. "
Or something like that.